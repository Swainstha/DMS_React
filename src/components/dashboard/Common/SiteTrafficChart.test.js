import React from 'react';
import SiteTrafficChart from './SiteTrafficChart';
import {
    configure,
    shallow,
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const setup = (props = {}) => {
    return shallow( < SiteTrafficChart {...props}  />
    )
}

it("Renders without error",()=>{
    const wrapper= setup({height:350});
    expect(wrapper).toMatchSnapshot();


})