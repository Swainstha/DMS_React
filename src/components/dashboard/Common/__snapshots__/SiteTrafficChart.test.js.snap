// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without error 1`] = `ShallowWrapper {}`;
